src/lib/mina_base/permissions.ml

```diff
@@ -319,6 +319,7 @@ module Poly = struct
         ; set_token_symbol : 'controller
         ; increment_nonce : 'controller
         ; set_voting_for : 'controller
+        ; set_timing : 'controller
```

Comment:
Instead of adding a new permission, we want to have it hard-coded as `Signature`, per the discussions I've been having.